bach/tests/conftest.py

```diff
@@ -43,32 +43,51 @@
 from collections import defaultdict
 from enum import Enum
 from typing import Dict, List
+from urllib.parse import quote_plus
 
 import pytest
 from _pytest.fixtures import SubRequest
 from _pytest.main import Session
 from _pytest.python import Metafunc, Function
 from _pytest.config.argparsing import Parser
+from dotenv import dotenv_values
 from sqlalchemy import create_engine
 from sqlalchemy.engine import Engine
 
+# Load settings from .test_env file, but allow overrides from Environment variables
+_DOT_ENV_FILE = os.path.dirname(os.path.dirname(os.path.realpath(__file__))) + '/.secrets/.test_env'
+_ENV = {
+    **dotenv_values(_DOT_ENV_FILE),
+    **os.environ
+}
 
-DB_PG_TEST_URL = os.environ.get('OBJ_DB_PG_TEST_URL', 'postgresql://objectiv:@localhost:5432/objectiv')
-DB_BQ_TEST_URL = os.environ.get('OBJ_DB_BQ_TEST_URL', 'bigquery://objectiv-snowplow-test-2/bach_test')
-DB_BQ_CREDENTIALS_PATH = os.environ.get(
+DB_PG_TEST_URL = _ENV.get('OBJ_DB_PG_TEST_URL', 'postgresql://objectiv:@localhost:5432/objectiv')
```

Comment:
is only used in this file
```suggestion
_DB_PG_TEST_URL = _ENV.get('OBJ_DB_PG_TEST_URL', 'postgresql://objectiv:@localhost:5432/objectiv')
```

Reply:
Done here: [cedc294](https://github.com/objectiv/objectiv-analytics/pull/1111/commits/cedc294b6ecdd8f02791d5fb2a02f8ef625a292d)